packages/style-engine/class-wp-style-engine-css-declarations.php

```diff
@@ -143,9 +137,9 @@ public function get_declarations_string( $should_prettify = false, $indent_count
 		$indent              = $should_prettify ? str_repeat( "\t", $indent_count ) : '';
 		$suffix              = $should_prettify ? ' ' : '';
 		$suffix              = $should_prettify && $indent_count > 0 ? "\n" : $suffix;
+		$spacer              = $should_prettify ? ' ' : '';
 
 		foreach ( $declarations_array as $property => $value ) {
-			$spacer               = $should_prettify ? ' ' : '';
```

Comment:
Moved this outside the loop. The `$spacer`'s value doesn't change on each iteration of the loop so it can be defined outside it.